src/helpers/findIdHelper.ts

```diff
@@ -0,0 +1,5 @@
+import { Tab } from '../types/Tabs';
+
+export function findId(tabs: Tab[], selectedTabId: string): Tab {
+  return tabs.find(({ id }) => id === selectedTabId) || tabs[0];
```

Comment:
Useless return statement

Reply:
I don't get why is it useless if it is not void function?